Fails to run on Windows 11

Clicked install from Zed Extensions panel and it comes back with an error that it failed to run.

This looks like it may be a path issue, at least based on the logs repeating C:\C:\

```
Language server cspell:

initializing server cspell, id 6
-- stderr--
node:internal/modules/cjs/loader:1222
  throw err;
  ^

Error: Cannot find module 'C:\C:\Users\nathany\AppData\Local\Zed\extensions\work\cspell\node_modules\.bin\cspell-lsp'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:158:5)
    at node:internal/main/run_main_module:30:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v22.5.1
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -4,7 +4,7 @@ use zed::settings::LspSettings;
 use zed_extension_api::{self as zed, LanguageServerId, Result};
 
 const NPM_PKG_NAME: &str = "cspell-lsp";
-const LS_BIN_PATH: &str = "node_modules/.bin/cspell-lsp";
+const LS_BIN_PATH: &str = "node_modules/cspell-lsp/dist/bundle.mjs";
 
 #[derive(Default)]
 struct CSpellExtension {
@@ -72,8 +72,7 @@ impl zed::Extension for CSpellExtension {
         Ok(zed::Command {
             command: zed::node_binary_path()?,
             args: vec![
-                env::current_dir()
-                    .unwrap()
+                zed_ext::sanitize_windows_path(env::current_dir().unwrap())
                     .join(server_path)
                     .to_string_lossy()
                     .to_string(),
@@ -99,3 +98,25 @@ impl zed::Extension for CSpellExtension {
 }
 
 zed::register_extension!(CSpellExtension);
+
+/// Extensions to the Zed extension API that have not yet stabilized.
+mod zed_ext {
+    /// Sanitizes the given path to remove the leading `/` on Windows.
+    ///
+    /// On macOS and Linux this is a no-op.
+    ///
+    /// This is a workaround for https://github.com/bytecodealliance/wasmtime/issues/10415.
+    pub fn sanitize_windows_path(path: std::path::PathBuf) -> std::path::PathBuf {
+        use zed_extension_api::{current_platform, Os};
+
+        let (os, _arch) = current_platform();
+        match os {
+            Os::Mac | Os::Linux => path,
+            Os::Windows => path
+                .to_string_lossy()
+                .to_string()
+                .trim_start_matches('/')
+                .into(),
+        }
+    }
+}

